force-app/main/default/classes/SOQL.cls

```diff
@@ -28,9 +28,10 @@ public with sharing class SOQL {
      * @return List of all leads with Id, LastName, Company, Phone, Email
      * Update the corresponding test class and testGetAllLeads method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Lead> GetAllLeads () {
+        Lead[] myLead = [SELECT Id,LastName,Company,Phone,Email FROM Lead];
+        return myLead;
```

Comment:
Just a small thing, but generally we would leave spaces after the commas for easier reading and convention. It doesn't hurt anything, though.